platformio_orig.ini

```diff
@@ -132,4 +132,4 @@ upload_protocol = esptool
 upload_protocol = esptool
 build_type = debug
 platform = https://github.com/platformio/platform-espressif32.git#develop
-platform_packages = framework-arduinoespressif32 @ https://github.com/espressif/arduino-esp32.git
+platform_packages = framework-arduinoespressif32 @ https://github.com/espressif/arduino-esp32.git
```

Comment:
please remove this edit, it changes nothing (?)

Reply:
nothing, is correct checked.